docs/AboutUs.md

```diff
@@ -3,67 +3,59 @@ layout: page
 title: About Us
 ---
 
+# About Us
+
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
-You can reach us at the email `seer[at]comp.nus.edu.sg`
+You can reach us at the email `florenyu[at]comp.nus.edu.sg`
 
 ## Project team
 
-### Audrey Felicio Anwar
-
-<img src="images/johndoe.png" width="200px">
-
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
-
-* Role: Advisor
-
```

Comment:
Should we remove this? I don't think it is supposed to be removed. we do u guys think @Eugene-Ong-W-X @PangKuangWei @Ugholaf

Reply:
Honestly it's up to you guys haha. The guide says it's ok to add or to not add. I erased it yesterday because I was too lazy to search for Audrey's photo, but shouldn't be too hard either if we want to add his photo. I remember it was somewhere in the ms teams doc.

Reply:
Oh is that so? Then I have no issues with the removal. Thanks for the clarification.